apps/wallet-mobile/src/yoroi-wallets/cardano/constants/sanchonet/constants.ts

```diff
@@ -0,0 +1,108 @@
+import {Balance} from '@yoroi/types'
+
+import {DefaultAsset} from '../../../types'
+import {COIN_TYPE, COINS_PER_UTXO_WORD, KEY_DEPOSIT, LINEAR_FEE, MINIMUM_UTXO_VAL, POOL_DEPOSIT} from '../common'
+
+export * from '../common'
+
+export const NETWORK_ID = 4_50
```

Comment:
`450` is fine, plenty of `isMainnet` checks that are considering `300` only will require update.